resources/zib/zib-BodyHeight.xml

```diff
@@ -0,0 +1,170 @@
+      <patternCoding>
+        <system value="http://loinc.org" />
+        <code value="8302-2" />
+      </patternCoding>
+    </element>
+    <element id="Observation.effective[x]">
+      <path value="Observation.effective[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.effective[x]:effectiveDateTime">
+      <path value="Observation.effective[x]" />
+      <sliceName value="effectiveDateTime" />
+      <short value="HeightDateTime" />
+      <definition value="Date and if possible time of body height measurement." />
+      <alias value="LengteDatumTijd" />
+      <type>
+        <code value="dateTime" />
+      </type>
+      <mapping>
+        <identity value="zib-bodyheight-v3.1.1-2020EN" />
+        <map value="NL-CM:12.2.4" />
+        <comment value="HeightDateTime" />
+      </mapping>
+    </element>
+    <element id="Observation.value[x]">
+      <path value="Observation.value[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.value[x]:valueQuantity">
+      <path value="Observation.value[x]" />
+      <sliceName value="valueQuantity" />
+      <short value="HeightValue" />
+      <definition value="The element contains the value of the measured body height.  &#xD;&#xA;Body height is the height of the human body from heel to scalp, generally measured when standing. Body height is measured in a stretched, horizontal position for very small children or patients that are unable to stand." />
+      <alias value="LengteWaarde" />
+      <type>
+        <code value="Quantity" />
+      </type>
+      <mapping>
+        <identity value="zib-bodyheight-v3.1.1-2020EN" />
+        <map value="NL-CM:12.2.2" />
+        <comment value="HeightValue" />
+      </mapping>
+    </element>
+    <element id="Observation.value[x]:valueQuantity.system">
+      <path value="Observation.value[x].system" />
+      <patternUri value="http://unitsofmeasure.org" />
+    </element>
+    <element id="Observation.value[x]:valueQuantity.code">
+      <path value="Observation.value[x].code" />
+      <binding>
+        <strength value="required" />
+        <description value="BodyLengthUnits" />
+        <valueSet value="http://hl7.org/fhir/ValueSet/ucum-bodylength" />
+      </binding>
+    </element>
+    <element id="Observation.note">
+      <path value="Observation.note" />
+      <short value="Comment" />
+      <definition value="Comment on the body height measurement." />
+      <alias value="Toelichting" />
+      <max value="1" />
+      <mapping>
+        <identity value="zib-bodyheight-v3.1.1-2020EN" />
+        <map value="NL-CM:12.2.3" />
+        <comment value="Comment" />
```

Comment:
This must be mapped on `Oberservation.note.text`, don't forget to add the know issues in the .yml file